reference/api/keys.md

```diff
@@ -68,19 +75,52 @@ List all existing API keys. **Expired keys are included in the response**, but d
       "createdAt": "2021-08-11T10:00:00Z",
       "updatedAt": "2021-08-11T10:00:00Z"
     }
-  ]
+  ],
+  "offset":0,
+  "limit":3,
+  "total":7
 }
 ```
 
+::: note
+API keys are displayed in descending order based on their `createdAt` date. This means that the most recently created keys appear first.
+:::
+
 ### Returned fields
 
+Returns API keys in an array called `results`, along with the following fields:
+
+#### `offset`
+
+The number of keys skipped over.
+
+#### `limit`
+
+The maximum number of keys to be returned by the request.
+
+#### `total`
+
+The total number of API keys that can be browsed.
+
+### The `results` array
+
+For each key, it returns:
+
+#### `name`
+
+A human-readable name for the key. Default value is `null`.
+
 #### `description`
 
 A description for the key. Default value is `null`.
 
+#### `uid`
+
+A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is generated by Meilisearch.
```

Comment:
I feel it's slightly more clear if we specify it's something that happens automatically.

```suggestion
A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is automatically generated by Meilisearch.
```